From 2039ad4dc228e65c9e3037975c10c84c400546da Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@gmail.com>
Date: Sun, 20 Jun 2021 15:25:56 -0700
Subject: [PATCH 128/196] Fix ticker symbol check in `abc_rpc_ecash.py` (#208)

Was missed due to other test failures during the merge ov v0.23.6.
This adjust the expectation to pass properly.
---
 test/functional/abc_rpc_ecash.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/functional/abc_rpc_ecash.py b/test/functional/abc_rpc_ecash.py
index 8cb03d41b..d96725243 100755
--- a/test/functional/abc_rpc_ecash.py
+++ b/test/functional/abc_rpc_ecash.py
@@ -53,7 +53,7 @@ class ECashRPCTest(BitcoinTestFramework):
         # is adapted to XEC only.
         # In BCHA mode, it triggers a "-fallbackfee is set very high!" error.
         self.restart_node(0, ["-ecash=0", "-fallbackfee=0"])
-        self.test_currency(ticker="Lotus",
+        self.test_currency(ticker="XPI",
                            satoshis_per_unit=1_000_000,
                            decimals=6)
 
-- 
2.39.5 (Apple Git-154)

